backend/app/views.py

```diff
@@ -83,3 +83,35 @@ def example_id(request, example_id):
     }
 
     return render(request, 'index.html', context)
+
+
+@api_view(['GET'])
+def all_documents(request):
+    doc_objs = Document.objects.all()
+    serializer = DocumentSerializer(doc_objs, many=True)
+    return Response(serializer.data)
+
+def documents(request):
+    """
+    All Documents page
+    """
+
+    context = {
+        'page_metadata': {
+            'title': 'Documents'
+        },
+        'component_name': 'Documents'
+    }
+
+    return render(request, 'index.html', context)
```

Comment:
Looks like `all_documents` and `documents` are copied over from PR #7, so let's remove them here or merge #7 in first.

Reply:
They've been removed